app/experimenter/static/scripts/experiment-list-select.js

```diff
@@ -1,3 +1 @@
-$("select[multiple]").selectpicker({
-  noneSelectedText: "All Types"
-});
+$("select[multiple]").selectpicker({});
```

Comment:
I wasn't able to get rid of this without making the filtering dropdowns disappear?